### PR TITLE
Base tier e2e tests

### PR DESCRIPTION
**Pull request by rajivnathan**
e2e tests for related PR: https://github.com/codeready-toolchain/host-operator/pull/398

Adds tests for the new base tier.

### CODE REVIEW

File: testsupport/tiers/checks.go [hunk 1]
```diff
@@ -43,6 +44,9 @@ var (
 
 func NewChecks(tier string) (TierChecks, error) {
 	switch tier {
+	case base:
```
Comment (alexeykazakov) [inline testsupport/tiers/checks.go:47]: Add basedeactivationdisabled too

Comment (rajivnathan) [reply testsupport/tiers/checks.go:48]: oops, done in https://github.com/codeready-toolchain/toolchain-e2e/pull/268/commits/6f1b6083896209ac78d010309b31dce485548146

**Review by alexeykazakov — APPROVED** [review]

**Review by MatousJobanek — APPROVED** [review]
:+1:

File: testsupport/wait/member.go [hunk 2]
```diff
@@ -380,6 +383,13 @@ func (a *MemberAwaitility) WaitForClusterResourceQuota(name string) (*quotav1.Cl
 		}
 		a.T.Logf("found ClusterResourceQuota '%s'", name)
 		quota = obj
+
+		for _, match := range criteria {
+			if !match(a, *obj) {
+				return false, nil
+			}
+		}
```
Comment (MatousJobanek) [inline testsupport/wait/member.go:391]: It's not critical, but I would add the check before it prints that the CRQ was found
```suggestion
		for _, match := range criteria {
			if !match(a, *obj) {
				a.T.Logf("Waiting until ClusterResourceQuota '%s' meets the given criteria", name)
				return false, nil
			}
		}
		a.T.Logf("found ClusterResourceQuota '%s'", name)
		quota = obj
```

**Review by MatousJobanek — APPROVED** [review]
Good solution with the CRQ test flakiness :+1:

**Pull request merged**